media/css/firefox/family/components/_hero.scss

```diff
@@ -20,71 +20,29 @@
         }
     }
 
-    .c-blurb {
-        background-color: f3.$green-light;
```

Comment:
**issue (blocking):** we want to keep this green bg color to match design (by default the blurb uses a purple color, we want to override that here)